src/Suave/Suave.fsproj

```diff
@@ -140,15 +179,7 @@
     <Reference Include="System.Web.Extensions" />
     <Reference Include="System.Xml" />
   </ItemGroup>
-  <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.5'">
-      <ItemGroup>
-        <Reference Include="FSharp.Core">
-          <HintPath>..\..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
-          <Private>True</Private>
-          <Paket>True</Paket>
-        </Reference>
-      </ItemGroup>
-    </When>
-  </Choose>
+  <Target Name="BeforeBuild" BeforeTargets="Build" Condition="'$(TargetPlatform)' == 'coreclr'">
+    <Exec Command="..\..\tools\nuget\NuGet.exe restore -Source https://www.myget.org/F/dotnet-core;https://www.myget.org/F/fsharp-daily;https://nuget.org/api/v2 project.json -PackagesDirectory ..\..\packages -Verbosity detailed" />

```

Comment:
Kevin, the point is we're trying to figure out what's needed in paket. Currently we didn't get good support from your colleagues. This pull request is pure gold since it creates a working F# app and I'm trying to understand what was needed to make it work. It's absolutely amazing stuff. 
My questions are no criticism to this pull request. It's the opposite.